nhrpd/nhrp_shortcut.c

```diff
@@ -401,9 +418,25 @@ static void nhrp_shortcut_send_resolution_req(struct nhrp_shortcut *s)
 		zb, NHRP_PACKET_RESOLUTION_REQUEST, &nifp->nbma,
 		&nifp->afi[family2afi(sockunion_family(&s->addr))].addr,
 		&s->addr);
-	hdr->u.request_id =
-		htonl(nhrp_reqid_alloc(&nhrp_packet_reqid, &s->reqid,
-				       nhrp_shortcut_recv_resolution_rep));
+
+
+	/* RFC2332 - The value is taken from a 32 bit counter that is incremented
+	 * each time a new "request" is transmitted.  The same value MUST
+	 * be used when resending a "request", i.e., when a "reply" has not been
+	 * received for a "request" and a retry is sent after an
+	 * appropriate interval
+	 */
+	if (!retry) {
```

Comment:
Although not completely consistent, FRR style normally omits the braces when there is a single conditional statement.